_episodes/56.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: episode
+title: "55: The vast possibilities of VAST + Trino"
+date: 2024-02-22
+tags: trino vast database catalog
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "Episode 56: The vast possibilities of VAST + Trino"
+  time: 0
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [Colleen Tartow](https://linkedin.com/in/colleen-tartow-phd), Field CTO and
+  Head of Strategy at [VAST Data](https://vastdata.com/).
+* [Roman Zeyde](https://www.linkedin.com/in/roman-zeyde/), Senior Software
+  Engineer at [VAST Data](https://vastdata.com/).
+
+## Release 439
+
+[Trino 439](https://trino.io/docs/current/release/release-439.html)
+
+* New caching layer for Delta Lake, Hive, and Iceberg!
+* Fix for setting session properties on catalogs with a `.` in the name.
+* Fix for reading Snappy data.
+
```

Comment:
Add

Trino 439 adds first documentation for new native file system support in Trino ( we should show doc page or at least link to it)

Trino Gateway 6 adds a docker container setup